src/scheduled-query/azext_scheduled_query/tests/latest/test_scheduled_query_scenario.py

```diff
@@ -49,13 +49,13 @@ def test_scheduled_query(self, resource_group):
                      self.check('criteria.allOf[0].failingPeriods.minFailingPeriodsToAlert', 1),
                      self.check('criteria.allOf[0].failingPeriods.numberOfEvaluationPeriods', 1)
                  ])
-        self.cmd('monitor scheduled-query create -g {rg} -n {name2} --scopes {rg_id} --condition "count \'union Event, Syslog | where TimeGenerated > ago(1h)\' > 360" --description "Test rule" --target-resource-type Microsoft.Compute/virtualMachines',
+        self.cmd('monitor scheduled-query create -g {rg} -n {name2} --scopes {rg_id} --condition "count \'union Event, Syslog | where TimeGenerated > ago(1h)\' > 360" --description "Test rule"',
```

Comment:
Same comment. As this is targeting VM in a resource group, you would add resource ID column.
Resource ID column is not manadatory, but not adding it will trigger alerts on the resource groups and not the VM.